### PR TITLE
Update readme, better wording

### PR DESCRIPTION
**Pull request by Bakhtiyar-Garashov**
# Change Summary

A small text change in the Readme in terms of wording.
Make method names snake_case and add docstring. Linter complains about not having docstring

## Description

Readme updated to have a better usage of english language in one statement. Also, method names converted to snake_case and docstring added to one of them

Fixes # (issue)
N/A

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/pyjdict/transformer.py [hunk 1]
```diff
@@ -11,25 +11,30 @@ class JdictTransformer(ast.NodeTransformer):
...
         visited_node = self.generic_visit(node)
         import_node = ast.ImportFrom(
-            module=jdict, names=[ast.alias(name=jdict)], level=0
+            module=JDICT, names=[ast.alias(name=JDICT)], level=0
         )
         visited_node.body.insert(0, import_node)
         return visited_node
 
-    def visit_Name(self, node: ast.Name) -> Any:
+    def visit_name(self, node: ast.Name) -> Any:
```
Comment (ShahriyarR) [inline src/pyjdict/transformer.py:27]: This syntax is from `ast` itself, i.e we have extended this from `NodeTransformer`.
Please check the link below, you will see the same pattern:
https://docs.python.org/3.9/library/ast.html#ast.NodeTransformer

File: src/pyjdict/transformer.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import ast
 from typing import Any, Final
 
-jdict: str = "jdict"  # WPS226 Found string literal over-use
+JDICT: Final = "jdict"  # WPS226 Found string literal over-use
```
Comment (ShahriyarR) [inline src/pyjdict/transformer.py:4]: Maybe `Final[str]`?

**Review by ShahriyarR — COMMENTED** [review]
First iteration

File: src/pyjdict/transformer.py (hunk 2 above)
Comment (Bakhtiyar-Garashov) [reply src/pyjdict/transformer.py]: Yeap, much better :) Fixed

File: src/pyjdict/transformer.py (hunk 1 above)
Comment (Bakhtiyar-Garashov) [reply src/pyjdict/transformer.py:27]: Ah, I see it. But it is interesting to see that kind of usage. I'd still suggest to stick to the common naming conventions of Python ecosystem which is snake_case.

**Comment by ShahriyarR** [comment]
@Bakhtiyar-Garashov Could you please pull recent changes for your main branch locally and run the formatter script?
`./scripts/format-imports.sh`
Then push it back)

**Pull request merged**